store/tikv/kv.go

```diff
@@ -165,7 +165,7 @@ func (s *tikvStore) CheckVisibility(startTime uint64) error {
 	}
 
 	if startTime < cachedSafePoint {
-		return ErrGCTooEarly
+		return ErrGCTooEarly.GenByArgs(startTime, cachedSafePoint)
```

Comment:
Are startTime and cachedSafePoint human readable?

Reply:
startTime is tso, and the safetime is calculated in `GCWorker.prepare`, also tso.